build/builder_test.go

```diff
@@ -33,6 +33,7 @@ func TestBuildv16(t *testing.T) {
 		t.Fatal(err)
 	}
 	b.indexTime, _ = time.Parse(time.RFC3339, "2018-07-06T18:44:45-07:00")
+	b.id = zoekt.BackfillID(b.indexTime, "github.com/a/b")
```

Comment:
> Rather than exporting BackfillID, can we read the v16 shard metadata and set it to the same value for it's id and indextime above?

done

> Once we have done that, we also don't need BackfillID to take a time, it can always use epoch

done